app/Http/Controllers/Api/ReportbackController.php

```diff
@@ -52,4 +56,31 @@ public function store(ReportbackRequest $request)
 
         return $this->item($reportback, $code);
     }
+
+    /**
+     * Update reportbackitem(s) that already exists.
+     *
+     * @param \Illuminate\Http\Request $request
+     * @return \Illuminate\Http\Response
+     */
+    public function updateReportbackItems(Request $request)
+    {
+        $items = $this->reportbackService->updateReportbackItems($request->all());
+
+        if (empty($items)) {
+            $code = 404;
+        } else {
+            $code = 201;
+        }
+
+        $meta = [];
+
+        if (count($items) === 1) {
+            $item = $items[0];
+
```

Comment:
@sbsmith86 is this ok to do? Looks a little strange but wasn't sure how to do a better way! Otherwise if I did a check in `ReportbackRepository.php` in the `updateReportbackItems` function, it'd be a lot of repeat code :-/